connectivity/check/deployment.go

```diff
@@ -953,33 +953,35 @@ func (ct *ConnectivityTest) deploy(ctx context.Context) error {
 			}
 		}
 
-		_, err = ct.clients.src.GetDeployment(ctx, ct.params.TestNamespace, echoExternalNodeDeploymentName, metav1.GetOptions{})
-		if err != nil {
-			ct.Logf("✨ [%s] Deploying echo-external-node deployment...", ct.clients.src.ClusterName())
-			containerPort := 8080
-			echoExternalDeployment := newDeployment(deploymentParameters{
-				Name:           echoExternalNodeDeploymentName,
-				Kind:           kindEchoExternalNodeName,
-				Port:           containerPort,
-				NamedPort:      "http-8080",
-				HostPort:       8080,
-				Image:          ct.params.JSONMockImage,
-				Labels:         map[string]string{"external": "echo"},
-				Annotations:    ct.params.DeploymentAnnotations.Match(echoExternalNodeDeploymentName),
-				NodeSelector:   map[string]string{"cilium.io/no-schedule": "true"},
-				ReadinessProbe: newLocalReadinessProbe(containerPort, "/"),
-				HostNetwork:    true,
-				Tolerations: []corev1.Toleration{
-					{Operator: corev1.TolerationOpExists},
-				},
-			})
-			_, err = ct.clients.src.CreateServiceAccount(ctx, ct.params.TestNamespace, k8s.NewServiceAccount(echoExternalNodeDeploymentName), metav1.CreateOptions{})
+		if ct.Features[features.NodeWithoutCilium].Enabled {
```

Comment:
Can we get a log line in `else` block that will indicate that we will not run these tests? I feel like this would save headache of whoever mistakenly tries to run these tests on a misconfigured cluster and won't know why they are not running.

Reply:
yeah sure any specific phrasing for the log message? maybe something like:
 
```
🔮 Skipping tests that require a node without Cilium
```
?

Reply:
ok added an info log:
```
ℹ️  Skipping tests that require a node Without Cilium
```